pkg/playground/react-demo-dialog.jsx

```diff
@@ -40,7 +40,7 @@
                         <tr>
                             <td className="top">
                                 <label className="control-label" for="control-1">
-                                    {_("Label")}
+                                    {"Label"}
```

Comment:
For your convenience :-) https://github.com/mvollmer/cockpit/commit/bfedddec918c8c1bc4f2772572aa75de9270c30b